academic-services/knowledge-exploration-service/introduction.md

```diff
@@ -1,16 +1,88 @@
+Once signed up for MAKES, new versions of the service and indexes will be delivered to you every 1-2 weeks.  Users have the option of using the default index supplied by the service, which includes all the entities and meta data included in MAG, or using a sub-graph of MAG to create their own custom indexes to power MAKES.  Users are also able to combine private data with the data from MAG to create custom indexes and search experiences for MAKES.
 
-## About Knowledge Exploration Service
+## Why use MAKES?
+
+Here are some key features of the Microsoft Academic Knowledge Exploration Service (MAKES):
+
+- *Self-hosting the Academic Knowledge APIs* - Many organizations have found great value in our [Academic Knowledge API](https://www.microsoft.com/research/project/academic-knowledge/) (formerly Cognitive Service APIs), but have been limited by the throughput allowed.  MAKES is a turn-key solution to self-host this set of APIs without limits.
+
+- *Create knowledge applications* - The set of MAKES APIs that are provisioned allow users to build dynamic knowledge applications that can analyze the state of research in particular areas, compare individuals and institutions and find objective connections between entities.
+
+- *Add a large corpus of academic entities to an existing search solution* - MAKES APIs can be federated to enhance existing search solutions allowing users access to a large corpus of academic research.
+
+- *Create a custom index from a subset of the Microsoft Academic Graph* - In some cases you may not need all of the data in MAG, or you would like combine your own private data with data from MAG.  MAKES allows you to generate an index MAG and private data.  An example would be to build an index specific to certain Journals or Institutions and include publications from your own organization.
+
+## What is MAKES?
+
+The Microsoft Academic knowledge Exploration Service (MAKES) in its' basic form is a self hosted REST API leveraging an index of all the entities in the Microsoft Academic Graph (MAG).  MAKES subscriptions deliver all the components needed to self host instances of this REST API and index.  By subscribing to MAKES, these components are delivered to your Azure subscription when new versions of MAG are created; typically once every 1 to 2 weeks.  Users of MAKES run a provided script and instances are automatically provisioned to their Azure account.
+
+The components shipped to a MAKES subscription include:
+
+- *MAKES Web Host Application* - This is the REST API web application that interacts with the MAKES index and grammar.
+
+- *MAKES VM Image* - A virtual machine image that is configured to support the MAKES web host application.
+
+- *MAKES Indexes* - Binary index files built from MAG that support the [Knowledge Exploration Service (KES)](#knowledge-exploration-service).
+
+- *MAKES Grammar* - A binary grammar file built to support academic related queries using [KES](#knowledge-exploration-service).
```

Comment:
"A binary grammar file built to support academic related queries using" => "MAKES includes a [SRGS](https://en.wikipedia.org/wiki/Speech_Recognition_Grammar_Specification) based language model to support the natural language query processing provided by its Interpret method."